### ISSUE TITLE
Reagent Bag Message does not go away

### ISSUE THREAD

**Issue by astrophel17** [AdiAddons/AdiBags#952]
### Addon Version

1.10.7

### World of Warcraft Version

Retail

### Describe the bug

Splash screen message for reagent bag (you've obtained a reagent bag! open your bag) appears upon every login, or when I craft a new one.  Opening the bag does not make it go away.  When I disable the add on and use the stock ui and open the bag, it goes away.  

### Steps to reproduce

log in.

### Verification

- [X] I have disabled all other addons and made sure this bug is triggered only with AdiBags enabled

**Comment by Talyrius** [comment]
Can anyone else corroborate experiencing this? Is the below no longer sufficiently silencing it?
https://github.com/AdiAddons/AdiBags/blob/c09e86e76c01e938ed6e2f0e898f6200a2697d14/core/Core.lua#L133-L138

**Comment by SpareSimian** [comment]
I'm still locked to 60 as I haven't bought DF. I decided to buy the 32-slot bags on the AH to replace the 20-slot ones I got during the pre-patch, and got the "you have a reagent bag!" popup when I picked up the new bag. I had to switch to standard bags and dismiss the popup to make it go away permanently. I just started a new alt into DF during the weekend trial and he did the quest that rewards a 26-slot bag. He got the same popup, even though he had the larger bag equipped. I just checked core.lua and see those lines so I'm definitely running that version.
